config/sync/core.entity_view_display.node.stanford_opportunity.default.yml

```diff
@@ -3,6 +3,7 @@ langcode: en
 status: true
```

Comment:
Some of the fields needs some small adjustments:
- Adjust the deadline to be more human readable
- Duplicate "eligibility" label
- Adjust "Start Date" to be more human readable
![image](https://github.com/user-attachments/assets/31e28f5b-e691-47d5-878e-01622116d23f)

- Duplicate "Prerequisites" label
![image](https://github.com/user-attachments/assets/52e50b9c-a167-42cc-8f23-ece964d5f9fb)